src/styles/main.scss

```diff
@@ -1,3 +1,63 @@
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+@for $i from 28 through 31 {
+  .calendar.month-length-#{$i} .calendar__day:nth-child(n + #{$i + 1}) {
+    display: none;
+  }
+}
+
+@for $i from 1 through 31 {
+  .calendar .calendar__day:nth-child(n + #{$i})::before {
```

Comment:
good job! :)
but I have some comments
here you can write just .calendar__day:nth-child... (not to increase specificity of selector)